src/ServiceManagement/ExpressRoute/Commands.ExpressRoute/BgpPeerings/RemoveAzureBGPPeering.cs

```diff
@@ -46,18 +49,35 @@ public override void ExecuteCmdlet()
                 ServiceKey.ToString(),
                 () =>
                 {
-                   
-                    if(!ExpressRouteClient.RemoveAzureBGPPeering(ServiceKey, AccessType))
+                    if (!PeerAddressType.HasValue || PeerAddressType == BgpPeerAddressType.All || AccessType != BgpPeeringAccessType.Microsoft)
```

Comment:
This code is not just for checking whether the right values for BgpPeeringAddressType has been specified. It is for deciding what the last parameter to the Remove API needs to be